base/math.jl

```diff
@@ -930,18 +930,28 @@ end
 ldexp(x::Float16, q::Integer) = Float16(ldexp(Float32(x), q))
 
 """
-    exponent(x::AbstractFloat) -> Int
+    exponent(x) -> Int
 
 Get the exponent of a normalized floating-point number.
+
 Returns the largest integer `y` such that `2^y ≤ abs(x)`.
```

Comment:
What if we revert the order of these sentences?

```suggestion
Returns the largest integer `y` such that `2^y ≤ abs(x)`.
For a normalized floating-point number `x` this corresponds to the exponent of `x`.
```

Or even remove the second sentence?